apps/system/js/task_manager.js

```diff
@@ -413,9 +421,19 @@ TaskManager.prototype = {
       idx = this.stack.length - 1;
     }
 
-    var currentPosition = this.scrollElement.scrollLeft;
     var desiredPosition = this.indexToOffset(idx);
 
+    if (immediately) {
+      this.scrollElement.scrollTo({
+        left: desiredPosition,
+        top: 0,
+        behavior: 'auto'
+      });
+      return Promise.resolve();
+    }
+
```

Comment:
does *reading* scrollLeft also cause reflow? I'd have thought it would always be precomputed/cached, but you know better than I.